fossevents/events/forms.py

```diff
@@ -0,0 +1,38 @@
+from django import forms
+
+from .models import Event
+from .services import send_confirmation_mail
+
+
+class EventForm(forms.ModelForm):
+    description = forms.CharField(widget=forms.Textarea(attrs={'placeholder': 'Enter description here Markdown'}),)
+    start_date = forms.DateTimeField(input_formats=["%Y-%m-%d", "%Y-%m-%dT%H:%M"],
+                                     widget=forms.DateTimeInput(attrs={'placeholder': 'YYYY-MM-DDTHH:MM',
+                                                                       'type': 'datetime-local'},
+                                                                format='%Y-%m-%dT%H:%M'))
+    end_date = forms.DateTimeField(input_formats=["%Y-%m-%d", "%Y-%m-%dT%H:%M"],
+                                   widget=forms.DateTimeInput(attrs={'placeholder': 'YYYY-MM-DDTHH:MM',
+                                                                     'type': 'datetime-local'},
+                                                              format='%Y-%m-%dT%H:%M'))
+
+    class Meta:
+        model = Event
+        exclude = ('auth_token', 'is_published')
+        widgets = {
+            'name': forms.TextInput(attrs={'placeholder': 'Title'}),
+            'homepage': forms.URLInput(attrs={'placeholder': 'http://example.com'}),
+            'owner_email': forms.EmailInput(attrs={'placeholder': 'user@example.com'}),
+        }
+
+    def clean_end_date(self):
+        start_date = self.cleaned_data.get('start_date')
+        end_date = self.cleaned_data.get('end_date')
+        if end_date and start_date:
```

Comment:
we are accessing 2 fields so this check should go to clean. that's the proper place when we access multiple fields.